src/core/mixins/handlerWallet.mixin.js

```diff
@@ -19,6 +19,9 @@ export default {
     getLatestPrices: {
       query: getLatestPrices,
       pollInterval: 600000,
+      skip() {
+        return !this.isEthNetwork;
```

Comment:
This is needed, no matter the network is, this is how we get the prices for the tokens so don't skip

Reply:
removed